shell/platform/darwin/macos/framework/Source/FlutterTextInputSemanticsObject.h

```diff
@@ -0,0 +1,87 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Cocoa/Cocoa.h>
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterPlatformNodeDelegateMac.h"
+
+#include "flutter/third_party/accessibility/ax/platform/ax_platform_node_base.h"
+
+@class FlutterTextField;
+@class FlutterTextInputPlugin;
+
+namespace flutter {
+
+//------------------------------------------------------------------------------
+/// The ax platform node for a text field.
+class FlutterTextPlatformNode : public ui::AXPlatformNodeBase {
+ public:
+  //---------------------------------------------------------------------------
+  /// @brief      Creates a FlutterTextPlatformNode that uses a
+  ///             FlutterTextField as its NativeViewAccessible.
+  /// @param[in]  delegate          The delegate that provides accessibility
+  ///                               data.
+  /// @param[in]  engine            The engine that is used for querying
+  ///                               the information about FlutterView and
+  ///                               FlutterViewController. The engine must
+  ///                               outlive the lifetime of this ax platform
+  ///                               node.
+  explicit FlutterTextPlatformNode(FlutterPlatformNodeDelegate* delegate, FlutterEngine* engine);
+  ~FlutterTextPlatformNode() override;
+
+  //------------------------------------------------------------------------------
+  /// @brief Gets the bounds of this platform node relative to the NSWindow. This
+  ///        is used by the FlutterTextField to get its frame rect.
+  NSRect GetWindowFrame();
```

Comment:
The comment and the name seem very different here. Is this getting a bounds, or a frame? Is it of the window, or of a node?